sample-apps/discount-functions-sample-app/extensions/order-discount/src/api.rs

```diff
@@ -15,102 +13,48 @@ pub mod input {
     #[derive(Clone, Debug, Deserialize)]
     #[serde(rename_all(deserialize = "camelCase"))]
     pub struct Input {
-        pub customer: Option<Customer>,
-        pub delivery_lines: Option<Vec<DeliveryLine>>,
-        pub locale: Option<String>,
-        pub merchandise_lines: Option<Vec<MerchandiseLine>>,
-        pub discount_node: Option<DiscountNode>,
+        pub discount_node: DiscountNode,
     }
 
     #[derive(Clone, Debug, Deserialize, Default)]
     pub struct DiscountNode {
-        pub metafield: Option<Metafield>,
+        pub metafield: Metafield,
```

Comment:
This should still be an `Option` AFAIK since there is no guarantee that the metafield exists